tensorflow/core/kernels/mkl_softmax_op.cc

```diff
@@ -61,15 +59,28 @@ class MklSoftmaxOp : public OpKernel {
       auto src_tf_shape = src_mkl_shape.IsMklTensor()
                               ? src_mkl_shape.GetTfShape()
                               : src_tensor.shape();
+      const int input_dims = src_tf_shape.dims();
       auto src_dims = TFShapeToMklDnnDims(src_tf_shape);
-      auto output_dims = src_dims;
+      memory::dims output_dims;
+      int axis;
+      if (src_mkl_shape.IsMklTensor()) {
+        axis = 1;
+        output_dims = src_mkl_shape.GetSizesAsMklDnnDims();
+      } else {
+        axis = input_dims - 1;
+        output_dims = src_dims;
+      }
       memory::format layout_type;
       // In MKL, data format passed to mkl softmax op depends on dimension of
-      // the input tensor. Here "x" data format in MKL is used for 1 dim tensor,
-      // "nc" for 2 dim tensor, "tnc" for 3 dim tensor, "nchw" for 4 dim tensor,
-      // and "ncdhw" for 5 dim tensor. Each of the simbols has the following
-      // meaning: n = batch, c = channels, t = sequence length, h = height, w =
-      // width, d = depth
+      // the input tensor.
+      // Here "x" data format in MKL is used for 1 dim tensor, "nc" for 2 dim
```

Comment:
Just curious, why do you insert newlines at the beginning of every sentence? Is this intentional?

Reply:
This is not intentional, it is generated by clang-format tool. Should this be changed back?

Reply:
I tried applying `clang-format` to the existing file and it didn't reformat this comment block. Could it be that you are using an old version of `clang-format`, or your `clang-format` has a different max-line-length other than 80? 

Could you please change this comment block back (but still fix `simbols` to `symbols`)? Thank you!